torch/_refs/__init__.py

```diff
@@ -522,6 +556,19 @@ def _neg_meta(a: TensorLikeType):
     extra_meta=_neg_meta,
 )
 
+
+@elementwise_type_promotion_wrapper(
+    type_promoting_args=("a,"),
+    type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.NO_OPMATH,
+)
+def positive(a: TensorLikeType) -> TensorLikeType:
```

Comment:
Add a comment for why this doesn't use _make_elementwise_unary_reference (because it doesn't support `out`, right?)